tests/Integration/WhatsAppCloudApiTest.php

```diff
@@ -301,6 +301,37 @@ public function test_send_reply_buttons()
         $this->assertEquals(false, $response->isError());
     }
 
+    public function test_send_reaction_message()
+    {
+        $this->markTestIncomplete(
```

Comment:
You can send first a message, store the message id from response and then send a reaction to that message

Reply:
https://github.com/derrickobedgiu1/whatsapp-cloud-api/commit/25622bff429d293f677ece2bc81c39e3586b3798